bHY6yh-de9900/cod-vyg67d_bHY6yh-de9900.xml

```diff
@@ -0,0 +1,162 @@
+           uestra. Si autem non dimiseritis homi<lb ed="#O" break="no"/>nibus 
+           nec pater uester dimittet pec<lb ed="#O" break="no"/>cata vestra. </quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e202">
+           <!--line bumps down-->
+           <ref xml:id="bHY6yh-de9900-Rd1e213">Marcus <!--line bumps up--> in euan. capitulo 11.</ref> ¶ <quote xml:id="bHY6yh-de9900-Qd1e206">Cum sta<!--line bumps back down-->bitis ad orandum dimittite 
+           <lb ed="#O"/>siquid habetis aduersus aliquem
+           <lb ed="#O"/>ut pater us qui in celis est dimittat 
+           <lb ed="#O"/>uobis peccata uestra.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e215">
+           <!--line bumps down-->
+           <ref xml:id="bHY6yh-de9900-Rd1e228">Lucas in euange<!--line bumps up-->lio capitulo 6.</ref> ¶ <quote xml:id="bHY6yh-de9900-Qd1e219">di<!--line bumps back down-->mittite et di<lb ed="#O" break="no"/>mittetur
+           uobis.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e225">
+           ¶ <ref xml:id="bHY6yh-de9900-Rd1e238">capitulo 17.</ref> <quote xml:id="bHY6yh-de9900-Qd1e227">Si pecca<lb ed="#O" break="no"/>uerit 
+           fratur tuus in te increpa illum
+           <lb ed="#O"/>et si paenitentiam egerit dimitte 
+           <lb ed="#O"/>illi et si septies in die conuersus fuerit 
+           <lb ed="#O"/>ad te dicens penitet me dimitte 
+           <lb ed="#O"/>illi.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e239">
+           <!--line bumps down-->
+           <ref xml:id="bHY6yh-de9900-Rd1e256">Io<!--line bumps up-->hannes in epistola. i. capitulo 1.</ref> ¶ <quote xml:id="bHY6yh-de9900-Qd1e243">Si con<!--line bumps back down-->fiteamur peccatam nostra fidelis et 
+           <lb ed="#O"/>iustus est dominus ut remittat nobis
+           <lb ed="#O"/>peccata nostra et emundet nos ab omni 
+           <lb ed="#O"/>iniquitate.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e251">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-de9900-Rd1e270">Iacobus apostolus <!--line bumps up--> epistola unica. capitulo 5. et fine</ref> <quote xml:id="bHY6yh-de9900-Qd1e255">Con<!--line bumps back down-->fiteamini alterutrum 
+           <lb ed="#O"/>peccata uestra.</quote>
+        </p>
+        <p xml:id="bHY6yh-de9900-d1e261">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-de9900-Rd1e282">Paulus apostolus <!--line bumps up--> epsitola ad Ro. capitulo 10. <!--line bumps back down--> </ref> ¶ <quote xml:id="bHY6yh-de9900-Qd1e265">Corde creditur 
+           ad iustitiam. ore autem confessio fit 
+           <lb ed="#O"/>ad salutem.</quote>
+        </p>
+           <cb ed="#O" n="b"/>
+        <p xml:id="bHY6yh-de9900-d1e271">
+           <lb ed="#O"/><ref xml:id="bHY6yh-de9900-Rd1e294"><sic>Psydorus</sic> in sinonima solilo<lb ed="#O" break="no"/>quiorum 
```

Comment:
column and page breaks always need to be inside a paragraph element. See fix below.

```suggestion
        </p>
        <p xml:id="bHY6yh-de9900-d1e271">
            <cb ed="#O" n="b"/>
           <lb ed="#O"/><ref xml:id="bHY6yh-de9900-Rd1e294"><sic>Psydorus</sic> in sinonima solilo<lb ed="#O" break="no"/>quiorum 
```